Sources/OutputParser.swift

```diff
@@ -808,4 +820,32 @@ class OutputParser {
 
         return nil
     }
+
+    private func parseExecutable(_ line: String) -> Executable? {
```

Comment:
Consider adding deduplication for executables by path, in case the same app is registered multiple times during build. Or document that duplicates are intentionally preserved.